website/members/models/member.py

```diff
@@ -5,13 +5,13 @@
 
 from django.conf import settings
 from django.contrib.auth.models import User, UserManager
-from django.db.models import Q
+from django.db.models import Q, Sum
 from django.urls import reverse
 from django.utils import timezone
 from django.utils.translation import gettext_lazy as _
 
 from activemembers.models import MemberGroup, MemberGroupMembership
-from payments.models import BankAccount
+from payments.models import BankAccount, Payment
```

Comment:
I would rather keep the `Payment` model out of the members application as much as possible.